app/models/flood_risk_engine/enrollment.rb

```diff
@@ -11,7 +11,7 @@ class Enrollment < ActiveRecord::Base
     # we query never from contact to its enrollment
     belongs_to :applicant_contact, class_name: "Contact"
     belongs_to :organisation
-    delegate :org_type, to: :organisation, allow_nil: true
+    delegate :org_type, :partners, to: :organisation, allow_nil: true
```

Comment:
Because you need to be able to get to partners from enrollment without having to check for organisation all the time.